app/models/move.rb

```diff
@@ -100,6 +93,26 @@ def current?
       (self.date.nil? && self.date_to.nil? && self.date_from.present? && self.date_from >= Time.zone.today)
   end
 
+  def prison_recall?
+    move_type == Move::move_types[:prison_recall]
+  end
+
+  def proposed?
```

Comment:
I think `enum` might give us those for free? https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html

Reply:
yes it does, thanks for spotting this 👍